src/demo-app/badge/badge-demo.html

```diff
@@ -0,0 +1,90 @@
+<div class="badge-demo">
+
+  <div class="badge-examples">
+    <h3>Text</h3>
+    <span [matBadge]="badgeContent" matBadgeOverlap="false" *ngIf="visible">
+      Hello
+    </span>
+
+    <span [matBadge]="11111" matBadgeOverlap="false">
+      Hello
+    </span>
+
+    <span matBadge="22" matBadgeOverlap="false" matBadgePosition="below after" matBadgeColor="accent">
+      Hello
+    </span>
+
+    <span matBadge="22" matBadgeOverlap="false" matBadgePosition="above before" matBadgeColor="warn">
+      Hello
+    </span>
+
+    <span matBadge="⚡️" matBadgeOverlap="false" matBadgePosition="below before">
+      Hello
+    </span>
+
+    <span [matBadge]="badgeContent" matBadgeDescription="I've got {{badgeContent}} problems">
```

Comment:
I think it’s preferable to use property binding `[matBadgeDescription]="..."`

Reply:
@jelbourn ^^?

Reply:
Using interpolation is fine when it's mixed with static text